stat/distuv/triangle.go

```diff
@@ -12,13 +12,13 @@ import (
 // Triangle represents a triangle distribution (https://en.wikipedia.org/wiki/Triangular_distribution).
 type Triangle struct {
 	a, b, c float64
-	Source  *rand.Rand
+	src     *rand.Rand
```

Comment:
Sorry, I should have been more clear; make it public and leave it out of the signature.